src/migrations/20241115143738-AddMonitoringEnum.js

```diff
@@ -0,0 +1,19 @@
+const { GOAL_CREATED_VIA } = require('../constants');
+const { prepMigration } = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      return Promise.all(Object.values(GOAL_CREATED_VIA).map((action) => queryInterface.sequelize.query(`
+         ALTER TYPE "enum_Goals_createdVia" ADD VALUE IF NOT EXISTS '${action}';
+      `)));
+    });
+  },
```

Comment:
THIS will fail in testing when GOAL_CREATED_VIA is updated in the future and will require updates to this migration file to fix. Constant values should be defined within the migration to avoid this.

Reply:
The test drop and recreate the DB, when we have a new value in GOAL_CREATED_VIA  for the next feature added. your migration will be the source of an error that will be hard to track down. I have had to go back and fix about a dozen other migration files for this same structure being used. Please change it.

Reply:
Ok I think what your saying here is instead of recreating all the enum's simply add a new one. This was done below per Matt's comment.